bfcfg

```diff
@@ -1393,14 +1678,15 @@ print_osarg()
 
 usage()
 {
-  echo "syntax: bfcfg [--help|-h] [--dump|-d] [--dump-osarg|o] [--part-info|-p] [--cfg2bin|-c <cfg_file>]"
+  echo "syntax: bfcfg [--help|-h] [--dump|-d] [--all|-a] [--dump-osarg|o] [--part-info|-p] [--cfg2bin|-c <cfg_file>]"
 }
 
 # Parse the arguments.
-options=$(getopt -n bfcfg -o dohpc: -l dump,dump-osarg,help,part-info,cfg2bin: -- "$@")
+options=$(getopt -n bfcfg -o adohpc: -l all,dump,dump-osarg,help,part-info,cfg2bin: -- "$@")
 eval set -- "$options"
 while [ "$1" != -- ]; do
   case $1 in
+    --all|-a) dump_all_mode=1 ;;
```

Comment:
Maybe we should add a dependency check here dump_mode != 0, and set dump_mode=2?